broker/test/test_broker/eventmesh.LockManager.spec.js

```diff
@@ -299,30 +298,27 @@ describe('eventmesh', () => {
             state: CONST.APISERVER.RESOURCE_STATE.UNLOCKED
           }
         };
-        mocks.apiServerEventMesh.nockPatchResource(CONST.APISERVER.RESOURCE_GROUPS.LOCK, CONST.APISERVER.RESOURCE_TYPES.DEPLOYMENT_LOCKS, 'samplelock6', samplelock6, 1, payload1);
-        return lockManager.unlock('samplelock6', samplelock6.metadata.resourceVersion)
+        const lockId = samplelock4.metadata.resourceVersion;
+        if (lockId) {
+          // update payload, if lockId is available
+          payload1.metadata.resourceVersion = lockId;
+        }
+        mocks.apiServerEventMesh.nockPatchResource(CONST.APISERVER.RESOURCE_GROUPS.LOCK, CONST.APISERVER.RESOURCE_TYPES.DEPLOYMENT_LOCKS, 'samplelock4', samplelock4, 1, payload1);
+        return lockManager.unlock('samplelock4', lockId)
           .then(() => {
             mocks.verify();
           });
       });
       it('should successfully unlock resource in first try if lock is not found', () => {
-        const payload1 = {
-          metadata: {
-            resourceVersion: samplelock6.metadata.resourceVersion
-          }
-        };
+        const payload1 = {};
         mocks.apiServerEventMesh.nockPatchResource(CONST.APISERVER.RESOURCE_GROUPS.LOCK, CONST.APISERVER.RESOURCE_TYPES.DEPLOYMENT_LOCKS, 'samplelock6', samplelock6, 1, payload1, 404);
```

Comment:
Since `lockId` (i.e `samplelock6.metadata.resourceVersion`) is `undefined`. The `metadata` will be skipped in request payload.